src/praisonai-agents/praisonaiagents/agent/agent.py

```
@@ -555,6 +556,7 @@ def __init__(
         backend: Optional[Any] = None,  # External managed agent backend (e.g., ManagedAgentIntegration)
         cli_backend: Optional[Union[str, Any]] = None,  # CLI backend for delegating turns (e.g., "claude-code")
         interrupt_controller: Optional['InterruptController'] = None,  # G2: Cooperative cancellation
+        sandbox: Optional[Union[bool, 'SandboxConfig']] = None,  # Sandbox for safe code execution
```

Ruff (0.15.12)

[error] Undefined name `SandboxConfig`

(F821)